torch/csrc/Size.cpp

```diff
@@ -1,6 +1,7 @@
 #include "Size.h"
 
 #include <string>
+#include "torch/csrc/utils/python_strings.h"
```

Comment:
I'm not sure why did we add this dir and why do we have `utils.h` and `utils/`. I have no idea what should go where. Can you explain it to me please? Do you want to split `utils.h` into a more fine grained set of headers?